src/pheval/analyse/generate_summary_outputs.py

```diff
@@ -96,53 +91,22 @@ def close(self) -> None:
             print("Error closing ", self.file)
 
 
-def generate_benchmark_gene_output(
-    prioritisation_data: TrackRunPrioritisation, plot_type: str
+def generate_benchmark_output(
+    prioritisation_data: TrackRunPrioritisation,
+    plot_type: str,
+    benchmark_generator: BenchmarkPrioritisationOutputGenerator,
```

Comment:
The `prioritisation_data` is only used by the `benchmark_generator` so why not just provide the specific `AnalysisResults` directly to the `BenchmarkPrioritisationOutputGenerator` or in this method rather than having a useless `TrackRunPrioritisation` class midway?